tests/readPreference/readpreference-ctor_error-006.phpt

```diff
@@ -1,18 +0,0 @@
---TEST--
-MongoDB\Driver\ReadPreference construction (invalid type for mode)
---FILE--
-<?php
-
-require_once __DIR__ . '/../utils/basic.inc';
-
-echo throws(function() {
-    new MongoDB\Driver\ReadPreference(3.14);
```

Comment:
Since we replaced `string|int` with `string` in the constructor signature, type coercion is now applied outside of strict mode and this would be implicitly cast to `'3.14'`. I didn't see any reason to use a different type (or declare strict types) just to assert a TypeError.